### DISCUSSION TITLE
Are there any publicly accessible instances of this?

### DISCUSSION THREAD

**Discussion by Scraggarax** [tphakala/birdnet-go#24] [Q&A]
I'm interested in trying it but I'd like to see it in action first to see what it's like compared to BirdNet-Pi. Anything I can look at to see the UI or how it operates?

**Comment by tphakala** [comment]
There is no UI yet in published releases, but I am working on one right now. I hope to publish first release with web ui soon, here is a quick demo on how first release will look..

https://github.com/tphakala/birdnet-go/assets/7030001/04b581a8-f874-4b70-a6a7-89e2e184beb0

**Reply by DMontgomery40** [reply]
@Scraggarax it's been about 3 months so not sure if you still want/need to see something public but here you go!

http://birdpi.drmont.com/

@tphakala feel free to share, I'd love to get all my birdnet-pi data in to see, well, more lol, but having some issues with the migration tool, I'll drop an issue in that repo tomorrow - and most importantly, THANK YOU!!  So much!  Let me know if I can do anything to help, that isn't code lol.  Testing, I've actually done a ton of mic testing over on the old repo (I have 3 pi's laying around somewhere, a domain that I'm happy to share more public instances on, etc).

**Reply by tphakala** [reply]
I don't recommend exposing BirdNET-Go to internet just yet, there is no security in place at all to protect configuration settings from unwanted changes etc. I do have plans to implement secure read only view for sharing to internet, there is lots of basic foundation to be build first like actual statistics etc.

But do I wonder why your daily summary grid is empty?

**Reply by DMontgomery40** [reply]
Okay I can pull it down tonight - but should it not look like this? 

![image](https://github.com/tphakala/birdnet-go/assets/130489651/4b468d23-55dc-440f-8712-40ee3562b78e)

**Reply by tphakala** [reply]
That looks okay, when I tried it daily summary was completely empty, only recent detections were visible.

Now that I think of it I bet it is a localisation issue with date picker.

**Reply by DMontgomery40** [reply]
ohhh is it March 5th over there now? yeah that would do it lol
